src/udev/udev-event.c

```diff
@@ -737,14 +687,13 @@ int udev_event_spawn(struct udev_event *event,
         if (r < 0)
                 return log_error_errno(r, "Failed to fork() to execute command '%s': %m", cmd);
         if (r == 0) {
-                /* child closes parent's ends of pipes */
-                outpipe[READ_END] = safe_close(outpipe[READ_END]);
-                errpipe[READ_END] = safe_close(errpipe[READ_END]);
+                if (rearrange_stdio(-1, outpipe[WRITE_END], errpipe[WRITE_END]) < 0)
+                        _exit(EXIT_FAILURE);
 
-                spawn_exec(event, cmd, argv, udev_device_get_properties_envp(event->dev),
-                           outpipe[WRITE_END], errpipe[WRITE_END]);
+                (void) close_all_fds(NULL, 0);
 
-                _exit(2);
+                execve(argv[0], argv, udev_device_get_properties_envp(event->dev));
+                _exit(EXIT_FAILURE);
```

Comment:
hmm, this changes the error code from 2 to 1 here... I hope this doesn't matter...